software/earl/authenticator.go

```diff
@@ -502,7 +502,7 @@ func (a *FileBasedAuthenticator) userHasAccess(user *User, target Target) (AuthR
 		if !isday {
 			return AuthOkButOutsideTime,
 				fmt.Sprintf("Fulltime user outside %d:00..%d:00",
-					hour_from, hour_to)
+					time_from, time_to)
```

Comment:
Maybe we should just add User.formatAccessTimes() string, as a convenience method.